q2_cutadapt/tests/test_demux.py

```diff
@@ -200,6 +200,27 @@ def test_batch_size_odd_number_of_samples(self):
         # obs_untrimmed should be empty, since everything matched
         self.assert_untrimmed_results(b'', obs_untrimmed_art)
 
+    def test_min_length(self):
+        metadata = CategoricalMetadataColumn(
+            # The third barcode is meant to completely remove the only GGGG
+            # coded sequence
+            pd.Series(['AAAA','CCCC','GGGGACGTACGT'], name='Barcode',
+                      index=pd.Index(['sample_a', 'sample_b', 'sample_c'],
+                      name='id')))
+
+        with redirected_stdio(stderr=os.devnull):
+            obs_demuxed_art, obs_untrimmed_art = \
+                self.demux_single_fn(self.muxed_sequences, metadata)
+
+        obs = obs_demuxed_art.view(SingleLanePerSampleSingleEndFastqDirFmt)
+
+        obs_f1, obs_f2 = obs.sequences.iter_views(FastqGzFormat)
```

Comment:
Since you are already referencing just the filepaths in your test assertions below, you could just unpack the values you need right off the bat.

```suggestion
        (obs_f1, _), (obs_f2, _) = obs.sequences.iter_views(FastqGzFormat)
```

Reply:
I knew there had to be a way to do that I just wasn't sure of the syntax. Thank you!